src/java/org/apache/cassandra/service/accord/api/AccordRoutingKey.java

```diff
@@ -204,8 +203,7 @@ public Range asRange()
 
         static
         {
-            Token key = getPartitioner().decorateKey(ByteBufferUtil.EMPTY_BYTE_BUFFER).getToken();
-            EMPTY_SIZE = ObjectSizes.measureDeep(new TokenKey(null, key));
+            EMPTY_SIZE = ObjectSizes.measure(new TokenKey(null, null));
```

Comment:
hah this change will allow several tests to avoid setting up DatabaseDescriptor...

Reply:
:)